mountpoint-s3/src/cli.rs

```diff
@@ -639,15 +636,26 @@ where
 
     let prefetcher_config = Default::default();
 
-    if let Some(path) = args.cache {
-        let metadata_cache_ttl = args.metadata_ttl.unwrap_or(Duration::from_secs(1));
-        filesystem_config.cache_config = CacheConfig {
-            serve_lookup_from_cache: true,
-            dir_ttl: metadata_cache_ttl,
-            file_ttl: metadata_cache_ttl,
-            ..Default::default()
-        };
+    let mut metadata_cache_ttl = args.metadata_ttl.unwrap_or_else(|| {
+        if args.cache.is_some() {
+            // When the data cache is enabled, use 1min as metadata-ttl.
+            TimeToLive::Duration(Duration::from_secs(60))
+        } else {
+            TimeToLive::Minimal
+        }
+    });
+    if matches!(metadata_cache_ttl, TimeToLive::Duration(Duration::ZERO)) {
+        const ZERO_TTL_WARNING: &str = "The '--metadata-ttl 0' setting is no longer supported, is now interpreted as 'minimal', and will be removed in a future release. Use '--metadata-ttl minimal' instead";
+        tracing::warn!("{}", ZERO_TTL_WARNING);
+        if !args.foreground {
+            // Ensure warning is visible even when not redirecting logs to stdout.
+            eprintln!("{}", ZERO_TTL_WARNING);
```

Comment:
What do you think about adding some color here just to make it stand out a bit for interactive users? I did this before for Shuttle with the `owo_colors` crate: https://github.com/awslabs/shuttle/blob/623c3b5a309f8537990b26987fe3410b2c710f0e/src/sync/atomic/mod.rs#L88-L95